tests/integration/_experimental/object_detection/test_evaluator_multiclass_fixed.py

```diff
@@ -642,20 +599,15 @@
             ignored=False,
             max_confidence_above_t=0.9,
             min_confidence_above_t=0.6,
-            thresholds=[ScoredClassificationLabel("e", 0.5)],
-            inference_labels=["b", "e"],
+            thresholds=[ScoredClassificationLabel("b", 0.5), ScoredClassificationLabel("e", 0.5)],
```

Comment:
Why were these not caught before? 

Should `assert test_sample_metrics == EXPECTED_COMPUTE_TEST_SAMPLE_METRICS` line be checking for the thresholds?

Reply:
There is a rough edge with the prebuilt OD integration tests being missed in the CI - investigating.